multi-node/aws/pkg/cluster/stack_template.go

```diff
@@ -223,6 +224,7 @@ func StackTemplateBody(defaultArtifactURL string) (string, error) {
 				map[string]interface{}{"IpProtocol": sgProtoUDP, "FromPort": 0, "ToPort": sgPortMax, "CidrIp": sgAllIPs},
 			},
 			"SecurityGroupIngress": []map[string]interface{}{
+				map[string]interface{}{"IpProtocol": sgProtoICMP, "FromPort": 3, "ToPort": -1, "CidrIp": sgAllIPs},
```

Comment:
Depends on who you ask. Some people don't like allowing echo request for example. Mostly with the argument that it can be used to waste server bandwidth (DoS).